packages/bsky/src/subscriptions/util.ts

```diff
@@ -0,0 +1,12 @@
+import { Subscriber } from './revenueCat'
+
+export const entitlementIdentifiersFromSubscriber = (
```

Comment:
I think we can make our revenue cat client a bit smarter and just move this logic into the `getSubscriber` method. Unless there's other info on response that we need elsewhere